src/tasklistClass.js

```diff
@@ -24,6 +30,7 @@ class Tasks {
   #addEvents = (id, li) => {
     const deleteButton = li.querySelector(`#delete-${id}`);
     const inputText = li.querySelector(`#input-${id}`);
+    const statusCompletion = li.querySelector(`#status-${id}`);
 
     li.addEventListener('dragstart', (e) => {
```

Comment:
This code could be removed as it's not updating the indexes of the objects stored after tasks are dragged and drop